### ISSUE TITLE
NJ: duplicate votes

### ISSUE THREAD

**Issue by pqnelson** [openstates/openstates#1709]
Gail Phoebus voted "no" and "other" for a vote [Reported out of Assembly committee with amendments](https://openstates.org/nj/votes/NJV00023908/), which seems incorrect.

I'm uncertain what's causing this error, otherwise I would endeavour to help.

**Addendum.** After some more digging, it appears to be a committee vote. The raw data comes from the [CA2016-2017](ftp://www.njleg.state.nj.us/votes/CA2016-2017.zip) votes, specifically lines 3837 _et seq._ which read:

```
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","Vainieri Huttle, Valerie","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","Tucker, Cleopatra G.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","Jones, Patricia Egan","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","Phoebus, Gail","R","Abstain"
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","Howarth, Joe","R","Abstain"
"A",2361,"Disab.-proh. snowplow block access",2/8/2016 13:00:00,"AHU","M",0,"r/Aca","McKnight, Angela V.","D","Yes"

"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Conaway, Herb","D",""
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Burzichelli, John J.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Lampitt, Pamela R.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Schaer, Gary S.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Spencer, L. Grace","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Quijano, Annette","D",""
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","DiMaio, John","R","No"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Schepisi, Holly","R","No"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Mosquera, Gabriela M.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Lagana, Joseph A.","D","Yes"
"A",2361,"Disab.-proh. snowplow block access",4/4/2016 13:00:00,"AAP","M",0,"r/Aca","Phoebus, Gail","R","No"
```

Note I added a linebreak, because there are really 2 votes occurring, which appears to be conflated into one by the crawler. (Observe the dates of the votes differ, one is 4/4/2016 whereas the other is 2/8/2016.)

**Comment by jamesturk** [comment]
thanks for the report :+1:

**Comment by jmcarp** [comment]
This should be fixed in #1766. Good to close?

**Comment by jamesturk** [comment]
👍
